packages/react-components/src/components/SendBox.tsx

```diff
@@ -105,6 +112,14 @@ export interface SendBoxProps {
    * boolean to determine if the input box has focus on render or not.
    */
   autoFocus?: 'sendBoxTextField' | false;
+  /* @conditional-compile-remove-from(stable): FILE_SHARING */
+  /**
+   * Optional callback to render uploaded files in the SendBox.
```

Comment:
Let's also document the behavior of sendbox when someone uses this.
For example, sendbox will grow veritcally to accomodate the height of injected component.
Injected component will be rendered below the text area in sendbox etc..

Reply:
done